### ISSUE TITLE
[Feature] Automatically generate studio covers

### ISSUE THREAD

**Issue by microdockman** [stashapp/stash#3188]
I manually added hundreds of studios, manually to make the cover workload too much, whether the function of automatic generating pictures or randomly grab the cover will be added in the later stage, just like generating ''markers'' gif or covers, it is very easy to use.
A few pictures are as follows

![001x](https://user-images.githubusercontent.com/31898637/204016368-9619b81e-2dd2-4ed2-9a21-f30a3cbb44ac.PNG)
![002x](https://user-images.githubusercontent.com/31898637/204016380-3bda56f7-bb37-4fb2-82dc-f5b128243942.PNG)
![003x](https://user-images.githubusercontent.com/31898637/204016394-3aaf327f-7016-4d24-a6b3-fee1faba73b5.PNG)

**Comment by DogmaDragon** [comment]
You seem to have a unique use case. Most people are not using studio images for covers, but for studio logos. 

Example
![image](https://user-images.githubusercontent.com/103123951/204067827-ab353256-326d-4abc-a900-c99d49f80988.png)

**Comment by microdockman** [comment]
> You seem to have a unique use case. Most people are not using studio images for covers, but for studio logos.
> 
> Example ![image](https://user-images.githubusercontent.com/103123951/204067827-ab353256-326d-4abc-a900-c99d49f80988.png)

well ,beautiful pictures  !!

I have multiple collections, nearly 200 girls. If I go to make the cover one by one, I might be weak. I just want to simply know who this collection is. 

Just like that ''markers'', very nice feature

In a similar way to the old problem that generated collections by folder, sometimes I didn't know which girl I was looking for, just randomly scan their covers

**Comment by bdbenim** [comment]
> ![001x](https://user-images.githubusercontent.com/31898637/204016394-3aaf327f-7016-4d24-a6b3-fee1faba73b5.PNG)

I notice that you've hidden the "Performers" page from your menu - isn't that what this is for?
![Screenshot 2022-12-01 132705](https://user-images.githubusercontent.com/97994155/205162610-8c574ee2-89eb-4077-929f-54f9d2a5b827.png)

> I just want to simply know who this collection is.

If your goal is to keep track of who is in each scene/gallery/image then you should be using performers. For example:
![Screenshot 2022-12-01 133339](https://user-images.githubusercontent.com/97994155/205163754-1c22a060-95f0-4664-81a0-b128a3585dd0.png)

Then you can use scrapers, such as stash-box, to download a photo for each one.

**Comment by microdockman** [comment]
> > ![001x](https://user-images.githubusercontent.com/31898637/204016394-3aaf327f-7016-4d24-a6b3-fee1faba73b5.PNG)
> 
> @microdockman, I notice that you've hidden the "Performers" page from your menu - isn't that what this is for? ![Screenshot 2022-12-01 132705](https://user-images.githubusercontent.com/97994155/205162610-8c574ee2-89eb-4077-929f-54f9d2a5b827.png)
> 
> > I just want to simply know who this collection is.
> 
> If your goal is to keep track of who is in each scene/gallery/image then you should be using performers. For example: ![Screenshot 2022-12-01 133339](https://user-images.githubusercontent.com/97994155/205163754-1c22a060-95f0-4664-81a0-b128a3585dd0.png)
> 
> Then you can use scrapers, such as stash-box, to download a photo for each one.




Yes, I have hidden the option which need fill details, because my file name is irregular, and I even don't know the specific names of some girls,(70%). If there is the function of automatic cover capture, it will not be too many requests to the network. Personally, I don't need a very beautiful cover, rough mark is ok
